core/controllers/base.py

```diff
@@ -399,18 +399,31 @@ def vaidate_and_normalize_args(self):
                 'Missing schema for %s method in %s handler class.' % (
                     request_method, handler_class_name))
 
-        normalized_value, errors = (
+        normalized_arg_values, errors = (
             payload_validator.validate(
                 handler_args, schema_for_request_method, extra_args_are_allowed)
         )
 
         self.normalized_payload = {
-            arg: normalized_value.get(arg) for arg in payload_arg_keys
+            arg: normalized_arg_values.get(arg) for arg in payload_arg_keys
         }
         self.normalized_request = {
-            arg: normalized_value.get(arg) for arg in request_arg_keys
+            arg: normalized_arg_values.get(arg) for arg in request_arg_keys
         }
 
+        # To accommodate the args, which are absent in request/payload
+        # but present in normalized_arg_values because these args are
+        # populated from their default_value by SVS architecture.
+        default_value_keys = list(
+            set(normalized_arg_values.keys()) -
+            set(payload_arg_keys + request_arg_keys)
+        )
+        for arg in default_value_keys:
```

Comment:
Add comment: "Populate the payload/request with the default args before passing execution onwards to the handler."

Reply:
Done